app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
```

Comment:
- [ ] Please make sure that views, the views in devise for the registration and login have been modified to match the wireframes and their styling:

Your layout  | Wireframe layout
------------- | -------------
<img width="1438" alt="Screenshot 2022-08-24 at 18 38 25" src="https://user-images.githubusercontent.com/77038610/186461460-e4626f4f-5677-4ae8-9d0a-472941cbdb17.png">  | <img width="248" alt="Screenshot 2022-08-24 at 18 39 04" src="https://user-images.githubusercontent.com/77038610/186461590-6e149e49-29ab-42c5-9a44-ca5a00077d48.png">